src/fairseq2/recipes/eval/__init__.py

```diff
@@ -26,6 +30,21 @@ def _add_wav2vev2_asr_eval_cli(group: CliGroup) -> None:
     )
 
 
+def _add_whisper_asr_eval_cli(group: CliGroup) -> None:
```

Comment:
This is highly redundant I think. We can try parameterize the evaluator setup function, the presets can be customized at runtime.